static/js/components/CommentList.jsx

```diff
@@ -4,6 +4,8 @@ import { useSelector, useDispatch } from "react-redux";
 import { Button } from "@material-ui/core";
 import Tooltip from "@material-ui/core/Tooltip";
 import GroupIcon from "@material-ui/icons/Group";
+
```

Comment:
```suggestion
```
and group all material-ui imports together (prop-types should probably go before those)